doc/ref/toc-osh.md

```diff
@@ -121,7 +121,8 @@ X [Unsupported]   enable
 </h2>
 
 ```chapter-links-osh-assign
-  [Literals]      sh-array      array=(a b c)   array[1]=B   "${a[@]}"
+  [Literals]      sh-init-list  a=([0]=1 [0]+=2 3)
+                  sh-array      array=(a b c)   array[1]=B   "${a[@]}"
```

Comment:
Maybe sh-init-list should also have the "value" like 

`a=(v [0]=1 ...)`

and maybe add a second example `a+=(v2 [c]=d [e]+=f)` or something

Reply:
> Maybe sh-init-list should also have the "value" like
> 
> `a=(v [0]=1 ...)`

I already included the "value" form as the third item.

In 4c4f59333, I have moved the third item to the first and adjusted the *value* of the values, indices, and keys.

> and maybe add a second example `a+=(v2 [c]=d [e]+=f)` or something - to make it clear that all the combinations do something

Yeah, that's a good idea. I added it in 4c4f59333.